TESTING.asciidoc

```diff
@@ -161,6 +161,11 @@ Don't count hypercores for CPU-intense tests and leave some slack
 for JVM-internal threads (like the garbage collector). Make sure there is
 enough RAM to handle child JVMs.
 
+You can set the default number of JVMs used in `~/.gradle/gradle.properties`:
```

Comment:
Maybe just reword so it is clear this is just for overriding the auto configured number of jvms.